xtflm_interpreter/src/dll_interpreter.cc

```diff
@@ -130,6 +130,12 @@ int get_output_tensor(inference_engine *ie, size_t tensor_index, void *value,
   return 0;
 }
 
+int get_input_tensor(inference_engine *ie, size_t tensor_index, void *value,
+                      const int size) {
```

Comment:
is there any reason you would want to pass size in here? rather than get it from get_tensor_details_buffer_sizes?

Reply:
get_tensor_details_buffer_sizes seems like a more complex method, size is passed in as it copies the implementation of get_output_tensor

Reply:
I removed tensor_arena_size as it was never used anywhere, maybe it would be good to keep it in case its used in the future?